__tests__/apiArticles.test.js

```diff
@@ -0,0 +1,66 @@
+const request = require("supertest");
+const app = require("../index");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data");
+const jestSorted = require('jest-sorted');
+
+// afterAll(() => {
+//   db.end();
+// });
+
+beforeEach(() => {
+  return seed(data);
+});
+
+describe("GET /api/articles", () => {
+  it("responds with an array of article objects", () => {
+    return request(app)
+      .get("/api/articles")
+      .then((response) => {
+        expect(response.status).toBe(200);
+        expect(response.body).toBeInstanceOf(Object);
+        expect(response.body.articles[0]).toHaveProperty("article_id");
+        expect(response.body.articles[0]).toHaveProperty("title");
+        expect(response.body.articles[0]).toHaveProperty("topic");
+        expect(response.body.articles[0]).toHaveProperty("created_at");
+        expect(response.body.articles[0]).toHaveProperty("votes");
+        expect(response.body.articles[0]).toHaveProperty("article_img_url");
+        expect(response.body.articles[0]).toHaveProperty("comment_count");
+        expect(response.body.articles[0].comment_count).toEqual(2);
+        expect(response.status);
+        // expect(response.body[0].created_at).toBeSorted({
```

Comment:
This will be needed, take a look at jest toBeSorted documentation for the array rather than the index.